.github/workflows/release-expo-cli-spec.yml

```diff
@@ -0,0 +1,27 @@
+name: 'Update expo-cli spec'
+on:
+  push:
+    tags:        
+      - 'expo-cli@*'
+  workflow_dispatch:
+
+jobs:
+  push-to-fig-autocomplete:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v2
+      - name: Generate the spec
+        run: |
+          yarn
+          yarn build
+          cd packages/expo-cli
+          yarn --silent introspect fig > new-spec.ts
+      - name: 'Create Autocomplete PR'
+        uses: withfig/push-to-fig-autocomplete-action@v1
+        with:
+          token: ${{ secrets.GH_TOKEN }}
```

Comment:
This requires a GH personal access token with repo scope that will be used to create a fork of the withfig/autocomplete repo and open a pull request against the main branch there.